### PR TITLE
modified for functional test

### PR DESCRIPTION
**Pull request by cxu2**
- added "name" for elements

### CODE REVIEW

File: template/changePassword.html [hunk 1]
```diff
@@ -22,7 +22,7 @@ <h1 style="color: #FFFFFF">Password Change</h1>
         <input class="form-control" type="password" placeholder="Confirm your new password" name = "passwordconfirm" required>
         <input class = "form-control" type="hidden" placeholder = "token" name = "token" value = "{{.Token}}">
     </div>
-    <button type="submit" class="btn btn-primary">Change Password</button>
+    <button type="submit" class="btn btn-primary" name="submit">Change Password</button>
 </form>
-</body>
+</body>s
```
Comment (DandilionLau) [inline template/changePassword.html]: Typo?

File: request_handler.go [hunk 2]
```diff
@@ -715,15 +709,11 @@ func logoutHandler(w http.ResponseWriter, r *http.Request) {
 
 //
 func main() {
-	flag.StringVar(&backendAddr, "backend", "localhost:8080", "backend IP and port")
-	flag.Parse()
-	backendURL = "http://" + backendAddr + backendURI
```
Comment (sleepingpig) [inline request_handler.go]: Why it's removed? I think it's good to have?

File: template/changePassword.html [hunk 3]
```diff
@@ -22,7 +22,7 @@ <h1 style="color: #FFFFFF">Password Change</h1>
         <input class="form-control" type="password" placeholder="Confirm your new password" name = "passwordconfirm" required>
         <input class = "form-control" type="hidden" placeholder = "token" name = "token" value = "{{.Token}}">
     </div>
-    <button type="submit" class="btn btn-primary">Change Password</button>
+    <button type="submit" class="btn btn-primary" name="submit">Change Password</button>
```
Comment (sleepingpig) [inline template/changePassword.html]: Why need to specify a name for it?

**Comment by sleepingpig** [comment]
I've carried the changes here to https://github.com/verizon-media-2019-ais3/identity-backend/pull/35

File: template/changePassword.html (hunk 3 above)
Comment (cxu2) [reply template/changePassword.html]: Just for selenium test's purpose, like findElement by name and then click on this element

**Pull request merged**